test/Dockerfile

```diff
@@ -62,10 +62,6 @@ RUN git clone https://github.com/folbricht/desync.git /tmp/desync && \
 RUN mkdir -p /lib/modules && \
     mkdir -p /var/run/dbus
 
-# We want to run as non-root user equaling uid of Travis' user 'travis' (2000)
-ENV user travis
+RUN adduser --disabled-password --gecos "" rauc-ci
```

Comment:
`man adduser` doesn't list `--gecos` anymore, but for `--comment` it says:

```man
       --comment comment
              Set  the  comment  field for the new entry generated.  adduser will not ask for the information if this option is given.  This field is also known under the name GECOS field and contains information that is used by the finger(1) com-
              mand.  This used to be the --gecos option, which is deprecated and will be removed after Debian bookworm.  Valid Modes: adduser, adduser --system.
```

Reply:
`--comment` is only supported from bookworm. At that point of the series, it's still bullseye.